checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -157,6 +164,26 @@ public NullnessAnnotatedTypeFactory(BaseTypeChecker checker, boolean useFbc) {
     }
 
     @Override
+    protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
```

Comment:
AbstractNullnessChecker takes a boolean whether to use raw/fbc.
Would it be more general to pass along that boolean to the Nullness ATF and use it here?